data-prepper-plugins/csv-processor/src/main/java/com/amazon/dataprepper/plugins/processor/csv/CSVProcessor.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.amazon.dataprepper.plugins.processor.csv;
+
+import com.amazon.dataprepper.metrics.PluginMetrics;
+import com.amazon.dataprepper.model.annotations.DataPrepperPlugin;
+import com.amazon.dataprepper.model.annotations.DataPrepperPluginConstructor;
+import com.amazon.dataprepper.model.event.Event;
+import com.amazon.dataprepper.model.processor.AbstractProcessor;
+import com.amazon.dataprepper.model.processor.Processor;
+import com.amazon.dataprepper.model.record.Record;
+
+import java.util.Collection;
+
+/**
+* Processor to parse CSV data in Events
+*
+* @since 2.0
+ */
```

Comment:
Please line up the asterisks. There is a standard Javadoc format for this. Here is an example.

https://github.com/opensearch-project/data-prepper/blob/31bf29b5c562ed74cecd65760c38c0a91edbca00/data-prepper-core/src/main/java/com/amazon/dataprepper/plugin/PluginProvider.java#L10-L16